src/commander/commander_btc.c

```diff
@@ -81,6 +82,13 @@ static commander_error_t _btc_pub_xpub(const BTCPubRequest* request, PubResponse
         default:
             return COMMANDER_ERR_GENERIC;
         }
+        if (n_written >= sizeof(title)) {
```

Comment:
n_written < 0 ||

Reply:
also move the whole thing to right after `n_written` is set. error handling should ideally be always the next line.

Reply:
Fixed.